agent-shell.el

```diff
@@ -84,7 +84,7 @@ returns the resolved path.  Set to nil to disable mapping."
   "Command prefix for executing commands in a container.
 
 When non-nil, both the agent command and shell commands will be
-executed using this runner. Should be a list of command arguments.
+executed using this runner.  Should be a list of command arguments.
```

Comment:
Let's remove unrelated changes from the PR please. Here and anywhere else below (they are mostly whitespace changes). If keen to do a cleanup, let's do it in a separate PR.